test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -263,24 +259,52 @@ pollWithRetry
     :: PactTestApiCmds
     -> ClientEnv
     -> RequestKeys
-    -> IO (Either ClientError PollResponses)
+    -> IO PollResponses
 pollWithRetry cmds env rks = do
   sleep 3
   go maxPollRetries
     where
-      go retries = do
-          result <- runClientM (pollApiCmd cmds (Poll (_rkRequestKeys rks))) env
+      go !retries = do
+          let retry failure = do
+                if retries == 0
+                  then do
+                    putStrLn $ "poll failing after " ++ show maxSendRetries
```

Comment:
`putStrLn`'s here and below?

Reply:
There are some already in this location, I didn't change that

Reply:
shouldn't this be logging?

Reply:
Tests shouldn't be chatty. (We have already too many of those.)

There are different ways to report test outputs such that it integrates with tasty's reporting.

The easiest for `IO` scenario tests is to use `testCaseSteps` which provides a "step" call back that logs messages as `tasty` progress reports messages.

Reply:
Oh, this is a golden test? Not sure if those provide a logging function.